kinetic/distribution.yaml

```diff
@@ -2343,6 +2343,22 @@ repositories:
       url: https://github.com/ensenso/ros_driver.git
       version: master
     status: developed
+  epson_g364_imu_driver:
+    release:
+      packages:
+      - epson_imu_driver
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://bitbucket.org/castacks/epson_g364_imu_driver-release.git
+      version: 0.0.2-0
+    doc:
```

Comment:
Travis is failing because the `doc` section should be before the `release` section to match alphabetical order